src/lib/searchPlaces.js

```diff
@@ -35,6 +35,37 @@ export type SearchResultCollection = {
   wheelmapFeatures?: (?WheelmapFeature)[] | Promise<?WheelmapFeature>[],
 };
 
+export function getOsmIdFromSearchResultProperties(
+  searchResultProperties?: SearchResultProperties
+) {
+  let osmId: ?number = searchResultProperties ? searchResultProperties.osm_id : null;
+
+  if (!osmId) {
+    return null;
+  }
+
+  // Only nodes with type 'N' and 'W' can be on Wheelmap.
+  if (searchResultProperties.osm_type !== 'N' && searchResultProperties.osm_type !== 'W') {
+    return null;
+  }
+
+  // Wheelmap stores features with osm type 'W' with negativ ids.
+  // @TODO Do this in some kind of util function. (Maybe wheelmap feature cache?)
+  if (searchResultProperties.osm_type === 'W') {
+    osmId = -osmId;
+  }
+
+  return osmId;
+}
+
+export function buildOriginalOsmId(searchResultProperties?: SearchResultProperties) {
+  if (!searchResultProperties) {
+    return 'osm://unknown/unknown';
```

Comment:
-1: Maybe I overlook something, but is there a reason why this is not `undefined`? As the string value's content is not typable, a developer that uses the `originalId` value at another end might overlook this case - if the only cases are '`undefined`' and 'defined + the value actually points to something', this could be safer.

Reply:
This will not happen in real life, I think our types are a bit lenient. Will change to undefined.